### PR TITLE
fix(DB/creature_template): TBC dungeon bosses damage output

### PR DESCRIPTION
**Pull request by Si1ker**
This PR will adjust the melee damage for TBC dungeon bosses based on the 'World of Warcraft Official strategy guide: Bestiary".

The current damage values in our DB ​​are incorrect as well as many other online DBs about them.
This book is the most official source of creatures data we can have nowadays.
All these changes have been taken from the official WoW Bestiary, they were adjusted and then tested manually one by one in-game. This process took several days but it will ensure all the changes are correct.

**Team project:**
Thanks to @Volgger for the Classic damage PR he posted a week ago and @BarbzYHOOL for all the help provided. We have been working as a team for a long time.

## CHANGES PROPOSED:
-  Update _mindmg_ and _maxdmg_ based on the official wow bestiary in _creature_template_
-  Update _DamageModifier_ for all creatures to 1 in _creature_template_ (this will allow better understanding and control on dmg values)

## ISSUES ADDRESSED:
- Continues the work done in #2742 

## TESTS PERFORMED:
- SQL changes implemented on last rev/hash, no errors found.
- Checked for old and new values, damage expected is correct.

## HOW TO TEST THE CHANGES:

- Import the SQL script and check that no error occurs during the implementation into the DB
- Go to any dungeon boss from this list and check for melee damage values before and after the update

## KNOWN ISSUES AND TODO LIST:

The following versions (normal and heroic) of TBC dungeon bosses were not found in the Bestiary and further research is required to update their damage values with official data.
- [ ] Shirrak the Dead Watcher 20318
- [ ] Exarch Maladaar 20306
- [ ] Pandemonius 20267
- [ ] Nexus-Prince Shaffar 20266
- [ ] Darkweaver Syth 20690
- [ ] Ambassador Hellmaw 20636
- [ ] Grandmaster Vorpil 20653
- [ ] Murmur 20657
- [ ] Mennu the Betrayer 19893
- [ ] Hydromancer Thespia 20629
- [ ] Mekgineer Steamrigger 20630
- [ ] Warlord Kalithresh 20633
- [ ] Hungarfen 20169
- [ ] Swamplord Musel'ek 20183
- [ ] Watchkeeper Gargolmar 18436
- [ ] Nazan 18432
- [ ] Vazruden 18434
- [ ] The Maker 18621
- [ ] Keli'dan the Breaker 18607
- [ ] Grand Warlock Nethekurse 20568
- [ ] Warbringer O'mrogg 20596
- [ ] Warchief Kargath Bladefist 20597
- [ ] Selin Fireheart 24723
- [ ] Selin Fireheart 25562
- [ ] Vexallus 24744
- [ ] Vexallus 25573
- [ ] Priestess Delrissa 24560
- [ ] Priestess Delrissa 25560
- [ ] Kael'thas Sunstrider 24857
- [ ] Lieutenant Drake 17848
- [ ] Lieutenant Drake 20535
- [ ] Captain Skarloc 17862
- [ ] Captain Skarloc 20521
- [ ] Zereketh the Unbound 21626
- [ ] Dalliah the Doomsayer 21590
- [ ] Wrath-Scryer Soccothrates 21624
- [ ] Harbinger Skyriss 21601
- [ ] Commander Sarannis 21551
- [ ] High Botanist Freywinn 21558
- [ ] Laj 21559
- [ ] Warp Splinter 17977
- [ ] Warp Splinter 21582
- [ ] Mechano-Lord Capacitus 21533
- [ ] Gatewatcher Gyro-Kill 21525
- [ ] Gatewatcher Iron-Hand 21526
- [ ] Nethermancer Sepethrea 21536
- [ ] Pathaleon the Calculator 21537
- [ ] Chrono Lord Deja 20738
- [ ] Temporus 20745
- [ ] Aeonus 20737

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Kitzunu** [comment]
I do not know how this guide works, but I would assume that it is values-based from retail boss damages. But are stats like armor that reduce the damage from bosses been taken into consideration?

What I am kind of asking is how the values were taken?

File: data/sql/updates/pending_db_world/rev_1584295889272307600.sql [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+-- The Black Morass (68 - 73)
+-- -------------------------------------------------------------------------------------------------------------------
+-- Chrono Lord Deja
+UPDATE `creature_template` SET `mindmg` = 1455, `maxdmg` = 2056, `DamageModifier` = 1 WHERE `entry` = 17879;
+-- Temporus
+UPDATE `creature_template` SET `mindmg` = 1829, `maxdmg` = 2587, `DamageModifier` = 1 WHERE `entry` = 17880;
+-- Aeonus
+UPDATE `creature_template` SET `mindmg` = 2353, `maxdmg` = 3327, `DamageModifier` = 1 WHERE `entry` = 17881;
+-- ===================================================================================================================
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1584295889272307600.sql]: Add a newline to the end of the file
```suggestion
-- ===================================================================================================================

```

**Comment by pklloveyou** [comment]
This PR was posted to the server last night, waiting for the actual test

**Comment by Si1ker** [comment]
@pklloveyou Thanks! 🚀
The only thing needed is that players or any tester from AC can confirm X boss is doing a damage value in the range of the change proposed. (By testing some few bosses, should be enough)
If the damage is higher than the maxdamage or lower than the mindmage, then a new commit from my side will be required.

I'll be waiting news from you ! Again, thank you 🐱

**Comment by pklloveyou** [comment]
My server is in the TBC phase

**Comment by FrancescoBorzi** [comment]
@pklloveyou any issues found so far? can we merge this?

**Comment by Si1ker** [comment]
This PR should not be merged or reopened as the work done does not resolve the issue.

**Pull request closed without merging**